### ISSUE TITLE
Research: Saving data to disk

### ISSUE THREAD

**Issue by MichielLangen** [wmeints/observatory#24]
Before replacing the way data is saved, from elasticsearch to disk, some research will be done to determine the best way to do this.

To make this possible, a way has to be found to store the data on the workstation of the user. For now, saving on the file system seems to be the best solution, but research will be done on this. The following points are important:

- Save data to disk, not using 3rd party software.
- Save efficiently
- Speed ​​is important

**Comment by MichielLangen** [comment]
To find the best solution for saving the data, I preformed a benchmark test. The results of this test are seen below. Six different saving methods where tested to see which was the fastest. The difference between JSON, Pickle and cPickle was very small. but with Pickle being the fastest we decided to go for that option.

![image](https://user-images.githubusercontent.com/28343855/54422567-18f06500-470f-11e9-8875-574f619d25bc.png)

  | Avegare (s)
-- | --
Zero measurement | 6,992
Pickle | 9,823
cPickle | 9,846
JSON | 9,886
File Saving | 10,377
PyTables | 19,662
SQLite | 60,936
